pipeline_template.yaml

```diff
@@ -0,0 +1,37 @@
+---
+version:
+
+date:
+
+pipelines:
+  - models:
+    model_name:
+    pipeline_name:
+    created_by:
+    creation_date:
+    version:
+    description:
+    job_name:
+    dataset:
```

Comment:
if were querying from DB after francoi is done, it wont necessarily have a name. Perhaps the query? or a description? in addition

Reply:
We could go with `dataset_description` to give an idea of what the dataset is.